src/backend/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -826,3 +826,34 @@ msgstr "Anglais"
 msgid "French"
 msgstr "Français"
 
+#: joanie/core/templates/issuers/contract_definition.html:22
+msgid "Template Contract Definition"
+msgstr "Convention de contrat"
+
+#: joanie/core/templates/issuers/contract_definition.html:28
+msgid "This document certifies that the student wants to enroll to the course"
+msgstr "Ce document certifie que l'étudiant souhaite suivre le cours"
+
+#: joanie/core/templates/issuers/contract_definition.html:33
+msgid "Your order is delivered by the organization"
+msgstr "Votre commande est délivrée par l'organisation"
+
+#: joanie/core/templates/issuers/contract_definition.html:42
+msgid "Student name"
+msgstr "Nom étudiant"
+
+#: joanie/core/templates/issuers/contract_definition.html:44
+msgid "Student address"
+msgstr "Adresse étudiant"
+
+#: joanie/core/templates/issuers/contract_definition.html:55
+msgid "Conditions of the contract"
+msgstr "Conditions de la convention"
+
+#: joanie/core/templates/issuers/contract_definition.html:66
+msgid "Signature of the student"
+msgstr "Signature de l'étudiant"
+
+#: joanie/core/templates/issuers/contract_definition.html:72
+msgid "Signature of the representative"
+msgstr "Signature du représentant"
```

Comment:
No need to manage translation in our PR. We are using crowding for that purpose and CI is in charge to upload then download and compile translations on each merge on the main branch.

Reply:
My bad, I will make sure next time to leave this section free since you have shown me how Crowdin takes care of new sentences to translate.